extension/src/experiments/data/index.ts

```diff
@@ -109,18 +109,12 @@ export class ExperimentsData extends BaseData<ExperimentsOutput> {
   }
 
   private async updateBranches() {
-    const [currentBranch, allBranches] = await Promise.all([
-      this.internalCommands.executeCommand<string>(
-        AvailableCommands.GIT_GET_CURRENT_BRANCH,
-        this.dvcRoot
-      ),
-      this.internalCommands.executeCommand<string[]>(
-        AvailableCommands.GIT_GET_BRANCHES,
-        this.dvcRoot
-      )
-    ])
+    const allBranches = await this.internalCommands.executeCommand<string[]>(
```

Comment:
[F] I realised that this did not make sense when digging further into the underlying calls/logic.

Reply:
I should have picked this up earlier.